buildSrc/subprojects/performance/performance.gradle.kts

```diff
@@ -3,8 +3,15 @@ apply(plugin = "org.gradle.kotlin.kotlin-dsl")
 dependencies {
     api(project(":integrationTesting"))
     implementation(project(":build"))
+    implementation("org.openmbee.junit:junit-xml-parser:1.0.0")
     implementation("org.codehaus.groovy.modules.http-builder:http-builder:0.7.2") {
         // Xerces on the runtime classpath is breaking some of our doc tasks
         exclude(group = "xerces")
     }
+    if (JavaVersion.current().isJava9Compatible) {
```

Comment:
Can't we use both dependencies on Java 8, too?

Reply:
So what is the problem if it conflicts with JDK 8? It is still compatible, right?

Reply:
Yes, it's compatible, so I was just concerning...